test/Functional/Controller/IndexController.php

```diff
@@ -20,4 +21,8 @@ public function indexAction(): ViewModel
 
         return $view;
     }
+
+    private function gtmDataLayer(array $options): void
```

Comment:
This is incorrect, but allows the static analysis to pass for now. I really need the correct implementation here. Any suggestions?

Reply:
This has been deleted now.